build-tests/api-documenter-test/etc/markdown/api-documenter-test.decoratorexample.creationdate.md

```diff
@@ -0,0 +1,23 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [api-documenter-test](./api-documenter-test.md) &gt; [DecoratorExample](./api-documenter-test.decoratorexample.md) &gt; [creationDate](./api-documenter-test.decoratorexample.creationdate.md)
+
+## DecoratorExample.creationDate property
+
+The date when the record was created.
+
+<b>Signature:</b>
+
+```typescript
+creationDate: Date;
+```
+<b>Decorators:</b>
+
+`@jsonSerialized`
```

Comment:
Shouldn't these link to the documentation for the decorators?

Reply:
They can, but the linking needs to be performed manually using `{@link}` tags.  Ideally API Extractor would analyze the decorator declaration and automatically link it, but the declaration doesn't actually appear in the .d.ts file, which is the reason for the `@decorator` workaround.